book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] We should be able to modify (set) the `rentals` from the Rental class as you have defined there, but here we can only read (get) the rental. `@rentals` should have a setter and getter, not just getter. Kindly add that.